lib/url.js

```diff
@@ -0,0 +1,3 @@
+const URL = '/api/questions';
```

Comment:
Could you try to use SystemJS to use ES6 modules on the client too? I can help with the configuration a bit to speed up the process.